src/modules/market-price/adapters/coingecko/index.ts

```diff
@@ -15,6 +15,7 @@ const symbolIdMap = {
   bal: "balancer",
   usdt: "tether",
   acx: "across-protocol",
+  snx: "havven",
```

Comment:
What a weird id

Reply:
Yea... I had to double-check because this seemed wrong to me